src/config/print/config.md

```diff
@@ -30,15 +17,31 @@ configure a printer using the command line.
 
 ## Graphical interface
 
-To configure the printer using a native GUI interface, run the following:
+The `system-config-printer` package offers simple and robust configuration of
+new printers. Install the package:
 
 ```
 # xbps-install -S system-config-printer
-# system-config-printer
 ```
 
-This command does not need to be run as root if PolicyKit is in use and
-`cups-pk-helper` is installed.
+Normally this tool would require root privileges to configure printers, but
+unprivileged access through `polkit` can be achieved with a helper. If you are
+interested in this, install the helper:
+
+```
+# xbps-install -S cups-pk-helper
+```
+
+You may run `system-config-printer` from your desktop environment's application
+launcher, or via the command line:
+
+```
+$ system-config-printer
```

Comment:
This should probably just be `# system-config-printer`. I'd like this a lot better if it was shown as running with root, followed by a note about the need to run as root, and the polkit exceptions and a mention of the `cups-pk-helper` package.

Reply:
Yeah I waffled a bit on whether to use root or user in the example. Went with user because it is typically bad form to run GUI apps as root. I will change it though. It is just an example, and as you mention below the audience will have some level of basic proficiency.